metaflow/plugins/aws/batch/batch_cli.py

```diff
@@ -149,6 +149,12 @@ def kill(ctx, run_id, user, my_runs):
     type=int,
     help="Number of parallel nodes to run as a multi-node job.",
 )
+@click.option(
+    "--secrets",
+    multiple=True,
```

Comment:
Does this apply to the `--with batch` syntax?

I haven't been able to get any of these to work:

```
python flow.py run --with batch:secrets=secret_a,arn:aws:secretsmanager:secret_a,secrets=secret_b,arn:aws:secretsmanager:secret_b
```

```
python flow.py run --with batch:secrets=secret_a,arn:aws:secretsmanager:secret_a;secret_b,arn:aws:secretsmanager:secret_b
```

Reply:
This isn't applicable anymore with the move to a dictionary-like type